exercises/error-handling/error_handling_test.go

```diff
@@ -27,6 +27,14 @@ func (mr mockResource) Close() error      { return mr.close() }
 func (mr mockResource) Frob(input string) { mr.frob(input) }
 func (mr mockResource) Defrob(tag string) { mr.defrob(tag) }
 
+// If this test fails and you've properly defined testVersion the requirements
```

Comment:
I wonder if this comment is necessary? I think a comment like this should either be in the `Hello-world` exercise as per #515 or should appear in all exercises rather than just randomly added in here to a later exercise.